examples/ibm-posture-management/README.md

```diff
@@ -0,0 +1,107 @@
+# Example for PostureManagementV1
```

Comment:
Let's have the example inside top-level ibm-scc directory

ibm-scc
 - findings (we already added that)
 - posture-management
 - admin 
etc to have all the scc things in one place